source/install/cloud-changelog.md

```diff
@@ -4,12 +4,36 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-08-10](#release-2023-08-10)
 - [Release 2023-08-03](#release-2023-08-03)
 - [Release 2023-07-26](#release-2023-07-26)
 - [Release 2023-07-20](#release-2023-07-20)
 - [Release 2023-07-19](#release-2023-07-19)
 - [Release 2023-07-11](#release-2023-07-11)
-- [Release 2023-06-26](#release-2023-06-26)
+
+## Release 2023-08-10
+
+### Improvements
+
+#### User Interface (UI)
+ - The number of channel members is now shown in the **Browse channels** modal.
+ - Updated the minimum required Edge version to 112+.
+ - An error is now displayed if a post edit history fails to load.
+ - Removed the deprecated Insights feature.
+ - Prepackaged Calls plugin version 0.18.0.
+ - Pre-packaged Playbooks version 1.38.0.
+
+### Bug Fixes
+ - Fixed an issue with missing zoneinfo in the Docker container.
+ - Fixed the error returned by ``PUT /api/v4/channels/{channelid}`` when the provided name already existed in the team.
+ - Fixed the clickable area of post textboxes being too small.
+ - Fixed a UI bug in the bot profile popover.
+ - Fixed an issue with the ``registerMessageWillBeUpdatedHook`` plugin hook.
+ - Fixed an issue where the **Saved Posts** section would not show channel and team names.
+
+### Known Issues
+ - Boards public links that follow the URL schema `/boards/public/...` no longer work. They can either be regenerated through the application by going to the board and selecting the **Share** button at the top right, or they can be obtained by replacing the `/boards/public/` part of the URL with `/plugins/focalboard/`.
+ - The Playbooks left-hand sidebar does not update when a user is added to a run or playbook without a refresh.
```

Comment:
I'm wondering if we should drop this? It's not likely we'll be fixing this issue anytime soon, especially with other known issues that we don't currently list here.